src/main/g8/src/main/scala/$package__packaged$/HelloWorldServer.scala

```diff
@@ -10,7 +10,7 @@ object HelloWorldServer extends IOApp {
 }
 
 object ServerStream {
-  def helloWorldService[F[_]: Effect] = new HelloWorldService[F].service
+  def helloWorldService[F[_]: Effect] = new HelloWorldService[F].routes
```

Comment:
Should the `def` still be called `_Service`, when it is now holding `HttpRoutes`?

Reply:
No, it probably shouldn't.  I'm not even that excited about it being a separate def.